addons/mail/views/mail_canned_response_views.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="mail_canned_response_view_search" model="ir.ui.view">
+            <field name="name">mail.canned_response.search</field>
+            <field name="model">mail.canned_response</field>
+            <field name="arch" type="xml">
+                <search string="Canned Responses Search">
+                    <field name="source"/>
+                    <field name="substitution"/>
+                </search>
+            </field>
+        </record>
+
+        <record id="mail_canned_response_action" model="ir.actions.act_window">
+            <field name="name">Canned Responses</field>
+            <field name="res_model">mail.canned_response</field>
+            <field name="view_mode">tree,form</field>
+            <field name="search_view_id" ref="mail_canned_response_view_search"/>
+            <field name="help" type="html">
+              <p class="o_view_nocontent_smiling_face">
+                Create a new canned response
+              </p><p>
+                Canned responses allow you to insert prewritten responses in
+                your messages by typing <i>:shortcut</i>. The shortcut is
+                replaced directly in your message, so that you can still edit
+                it before sending.
+              </p>
+            </field>
+        </record>
+
+        <record id="mail_canned_response_view_tree" model="ir.ui.view">
+            <field name="name">mail.canned_response.tree</field>
+            <field name="model">mail.canned_response</field>
+            <field name="arch" type="xml">
+                <tree string="Canned responses" editable="bottom" default_order="is_shared" sample="1">
+                    <field name="source" readonly="not is_editable"/>
```

Comment:
Should be managed with ACLs

Reply:
Record rules do not prevent edition UI in list views.